packages/flutter/lib/src/material/theme.dart

```diff
@@ -27,18 +28,18 @@ const Duration kThemeAnimationDuration = const Duration(milliseconds: 200);
 ///    than changing the theme all at once.
```

Comment:
What should we describe? I figured it might be confusing since behaviourally, icons still behave all the same way vs everything else that we're not calling out explicitly in the documentation from the user's perspective.

Or maybe you're suggesting to call out all the subset things in ThemeData that has its own InheritedWidget?